package.json

```diff
@@ -42,9 +44,11 @@
     "react-native-gesture-handler": "~1.10.2",
     "react-native-get-random-values": "~1.7.0",
     "react-native-paper": "^4.9.2",
+    "react-native-push-notification": "^8.0.0",
     "react-native-reanimated": "~2.2.0",
     "react-native-safe-area-context": "3.2.0",
     "react-native-screens": "~3.4.0",
+    "react-native-simple-toast": "^1.1.3",
```

Comment:
Instead of using this library which doesn't have a lot of support can we replace it with: https://callstack.github.io/react-native-paper/snackbar.html

Reply:
done